### PR TITLE
Issue #1768 ExecutionInput: allow to set value a top level

### PR DESCRIPTION
**Pull request by dimdimych**
Here's the push request it's a proposal for the issue #1768

### CODE REVIEW

**Comment by dimdimych** [comment]
Hm... It looks like some tests not related to my fix are failing

**Comment by andimarek** [comment]
Thanks for the PR @dimdimych. Test test are failing because your change, please have a look again. thanks

File: src/test/groovy/graphql/Issue1768.groovy [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        when:
+        ExecutionResult result = graphql.execute {
+            it.query(" { dummy } ")
+        }
+
+        then:
+        result.data.dummy == null
+
+    }
+    def "#1768 test if local context is set for top level"() {
```
Comment (andimarek) [inline src/test/groovy/graphql/Issue1768.groovy]: can you remove the reference to the issue number? thanks

File: src/test/groovy/graphql/Issue1768.groovy [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+        when:
+        ExecutionResult result = graphql.execute {
+            it.localContext("test").query(" { dummy } ")
+        }
+
+        then:
+        result.data.dummy == "test"
+    }
+    def "#1768 test if the top local context gets transferred to the next level"() {
```
Comment (andimarek) [inline src/test/groovy/graphql/Issue1768.groovy]: same here, please remove the reference to the issue

**Pull request merged**